source/class/qx/tool/cli/commands/Test.js

```diff
@@ -22,18 +22,16 @@ const process = require("process");
 /**
  * Compiles the project, serves it up as a web page (default, can be turned off),
  * and dispatches the "runTests" event. All tests that should be run need to
- * register an event handler. The handlers are called with a {@link qx.even.type.Data}
+ * register themselfs by the test command. 
+ * 
+ * The runTests event is called with a {@link qx.even.type.Data}
  * containing the command instance.
  *
  * A test can write to the (native) `errorCode` property of this instance,
  * which is used to determine the exit code of the `qx test` command.
  * This means, however, that the last run test overwrites the
  * `errorCode` of any previous test, and has to take care of this.
  *
- * A more scalable solution is to  call {@link
```

Comment:
Shouldn't it be the other way round - remove the docs on `errorCode` which is only there for BC purposes, and tell people to use the proper way with the `qx.tool.cli.api.Test`, since this is the cleaner solution which doesn't require boilerplate to check existing exitCodes? Also, this supports the `--fail-fast` option which `errrorCode` doesn't (without additional checks).

Reply:
I would still argue that we should encourage people to use the `exitCode` qooxdo property of the Test instance instead of the `errorCode` native property ...